tests/src/testhelpers.hh

```diff
@@ -59,3 +60,7 @@ void checkSolverInfos(TestSuiteType& t, const std::vector<int>& expectedIteratio
     t.check(controlInfo.solverInfos[i].success) << "Failed to converge at step " << i;
   }
 }
+
+inline auto testLocation(std::source_location loc = std::source_location::current()) {
```

Comment:
Where is this used I maybe missed it and if yes why?

Reply:
I am using it in the tests here
https://github.com/ikarus-project/ikarus/blob/5c4080c46901806686a167ab80a7d12d0565e3f4/tests/src/testvtkwriter.cpp#L99
This allows us to see where exactly the failed test is, I think its a neat feature

Reply:
Yes ok. Nice.